src/dashboard/pages/onboarding/OnboardingMetamask.vue

```diff
@@ -13,6 +13,14 @@ v-stepper(v-model="step", vertical)
     | Log in to MetaMask
   v-stepper-content(step=2)
     b Now open MetaMask to log in to your account.
+    div(class="note") #[b NOTE:] Unfortunately, #[a(href='https://github.com/MetaMask/metamask-extension/issues/5950') due to (temporary) technical reasons], you also need to #[b turn off privacy mode] in MetaMask (#[a(href='https://medium.com/metamask/https-medium-com-metamask-breaking-change-injecting-web3-7722797916a8') details on privacy mode]). You can do this by:
```

Comment:
Make the link contain less of the sentence, probably just the "technical reasons".

Reply:
Also, you can just do `div.note` instead of `div(class="note")`.